### PR TITLE
Create the ticket skill

### PR DESCRIPTION
**Pull request by gmartinon-ed**
### Description

- Introduces the `ticket` skill for Claude Code: a specialized agent that writes developer-ready GitHub tickets for the GLIDE dev team from any input (research paper, refactoring request, documentation task, or repository hygiene work).
- The skill explores the codebase, proposes architecture choices, writes concrete pseudocode, and produces a markdown file in `tickets/`.
- Adds four reference ticket templates (feature, documentation, refactoring, repository) used by the skill to structure its output consistently.
- Adds an eval suite (`evals.json`) with test cases to benchmark skill quality and triggering accuracy.
- Enables the `skill-creator` plugin in `.claude/settings.json` to allow creating and improving skills directly from the IDE.

### Type of change

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Refactoring (no behavior change)
- [ ] Documentation
- [x] Repository hygiene

### Checklist

- [x] I have read `CONTRIBUTING.md`
- [x] `make lint` passes
- [x] `make type-check` passes
- [x] `make tests` passes
- [x] `make coverage` reports 100% coverage
- [x] `make test-notebooks` passes
- [ ] New public API has numpy-style docstrings
- [ ] New public API is inserted in the API reference section of the documentation
- [ ] Docs build without warnings (`make doc`)
- [ ] `CHANGELOG.md` updated if the change is user-facing

### LLM usage

- [ ] No LLM used
- [x] LLM used: Claude Sonnet 4.6
- [x] I went through and validated all the code myself

### CODE REVIEW

File: .claude/skills/ticket/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+|---|---|
+| Feature | `references/template-feature.md` |
+| Refactoring | `references/template-refactoring.md` |
+| Documentation | `references/template-documentation.md` |
+| Repository | `references/template-repository.md` |
+
+Follow the template exactly. Acceptance criteria must be specific to this ticket. Never include items already covered by the team's PR template — the following are required on every PR and must not appear in ticket-specific criteria:
+
+- `make lint` passes
+- `make type-check` passes
```
Comment (imerad) [inline .claude/skills/ticket/SKILL.md]: Ajouter pointeur vers la PR template au cas où elle changerait ?

File: .claude/skills/ticket/SKILL.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+from typing import Dict, List, Optional, Tuple
+```
+
+**Return statements** — assign first, then return; never compute inside `return`:
+```python
+result = a * b + c
+return result
+# not: return a * b + c
+```
+
```
Comment (imerad) [inline .claude/skills/ticket/SKILL.md]: Idem, ce ne serait pas mieux de pointer vers un fichier qui fournit déjà ces conventions ? C'est dans CLAUDE.md

File: .claude/skills/ticket/references/template-feature.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    >>> estimator = ClassName(param=value)
+    >>> result = estimator.compute_mean_estimate(labeled, proxy)
+    1.5
+    """
+
+    def __init__(self, param: Type) -> None:
+        self.param = param
+
+    def compute_mean_estimate(self, labeled: np.ndarray, proxy: np.ndarray) -> float:
+        """One-line summary.
```
Comment (imerad) [inline .claude/skills/ticket/references/template-feature.md]: On utilise généralement NDArray pour ces annotations, vaut mieux ne pas induire en erreur avec np.ndarray ici

File: .claude/skills/ticket/references/template-feature.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+```
+
+[Repeat for each class or standalone function in scope.]
+
+## Functional tests
+
+[Include this section only when a mathematical equivalence exists — typically when the new feature is a generalisation of an existing one and the simpler case should be recoverable as a special case. Omit entirely otherwise.
+
+For each equivalence, state it plainly and show the corresponding test structure.]
+
```
Comment (imerad) [inline .claude/skills/ticket/references/template-feature.md]: Ça risque d'être limitant de restreindre aux équivalences avec un cas existant. Nous avons par exemple le StratifiedPPI qui doit être au moins aussi bon que PPI. Je recommande d'écrire quelque chose de plus générique ici comme "test for expected mathematical properties of the new object beyond code validity"

File: .claude/skills/ticket/references/template-feature.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+**[Name of the equivalence, e.g. "Single-stratum stratified PPI equals plain PPI"]**
+
+[One sentence: "When all observations belong to a single stratum, StratifiedPPIMeanEstimator must return the same value as PPIMeanEstimator."]
+
+```python
+def test_single_stratum_equals_ppi():
+    labeled = np.array([1.0, 2.0])
```
Comment (imerad) [inline .claude/skills/ticket/references/template-feature.md]: plutôt un vecteur de la même taille avec des NaNs dans labeled ?

File: .claude/skills/ticket/references/template-refactoring.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+## Corner cases
+
+- [Things that could break during the refactor — callers that may not be obvious, implicit contracts, test fixtures that need updating, public API changes that affect users]
+- [Another scenario]
+
+## Acceptance criteria
+
+- [ ] [Specific deliverable, e.g. "All estimators expose compute_mean_estimate rather than a class-specific alias"]
```
Comment (imerad) [inline .claude/skills/ticket/references/template-refactoring.md]: Plutôt `estimate()` qui correspond à la codebase ?

File: .claude/skills/ticket/references/template-repository.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+## Task breakdown
+
+- [ ] [Specific subtask — name the file to create or modify using full, self-explaining names (e.g. `compatibility.yml` not `compat.yml`, `spec0-matrix.yml` not `spec0.yml`)]
+- [ ] [Another subtask]
+- [ ] [...]
+
+## Acceptance criteria
+
+- [ ] [Specific and testable, e.g. "A PR that introduces a new public function without a docstring is blocked by CI"]
+- [ ] [Another specific deliverable, e.g. "Running `make pre-commit` on a notebook with outputs strips them before committing"]
```
Comment (imerad) [inline .claude/skills/ticket/references/template-repository.md]: Plutôt "Running `make pre-commit` strips all notebooks from outputs before committing"

**Review by imerad — CHANGES_REQUESTED** [review]

File: .claude/skills/ticket/SKILL.md (hunk 1 above)
Comment (gmartinon-ed) [reply .claude/skills/ticket/SKILL.md]: Fait

File: .claude/skills/ticket/SKILL.md (hunk 2 above)
Comment (gmartinon-ed) [reply .claude/skills/ticket/SKILL.md]: Done

File: .claude/skills/ticket/references/template-feature.md (hunk 3 above)
Comment (gmartinon-ed) [reply .claude/skills/ticket/references/template-feature.md]: Done

File: .claude/skills/ticket/references/template-feature.md (hunk 4 above)
Comment (gmartinon-ed) [reply .claude/skills/ticket/references/template-feature.md]: Done

File: .claude/skills/ticket/references/template-feature.md (hunk 5 above)
Comment (gmartinon-ed) [reply .claude/skills/ticket/references/template-feature.md]: Done

File: .claude/skills/ticket/references/template-refactoring.md (hunk 6 above)
Comment (gmartinon-ed) [reply .claude/skills/ticket/references/template-refactoring.md]: Done

File: .claude/skills/ticket/references/template-repository.md (hunk 7 above)
Comment (gmartinon-ed) [reply .claude/skills/ticket/references/template-repository.md]: Done

**Review by imerad — APPROVED** [review]
